doc/whats_new/v1.5.rst

```diff
@@ -537,6 +537,11 @@ Changelog
 - |Fix| :func:`~utils._safe_indexing` now works correctly for polars DataFrame when
   `axis=0` and supports indexing polars Series.
   :pr:`28521` by :user:`Yao Xiao <Charlie-XIAO>`.
+  
+- |Fix| :func:`utils._ovr_decision_function`
```

Comment:
We should move the entry in 1.6.

Also, the documentation should not be related to a private function but instead to a public class and method. So here, we have to track what potential we had. I assume the first impacted estimator is `OneVsOneClassifier` looking at the diff. But we should look at the other estimator using `utils._ovr_decision_function`.

Reply:
I understand, thank you. I will move the entry to 1.6. It appears that the main impacted estimator is `OneVsOneClassifier`, but there is also an impact on `svm`. In this case, should the entry still refer to the `sklearn.utils` module, or should it refer specifically to the modules of `OneVsOneClassifier` and `svm`?